docs/docusaurus/docs/reference/learn/glossary.md

```diff
@@ -3,6 +3,25 @@ id: glossary
 title: "Glossary"
 ---
 
-import InProgress from '../../core/_core_components/_in_progress.md'
+[Action](/core/trigger_actions_based_on_results/trigger_actions_based_on_results.md): A component that integrates Great Expectations with other tools based on a Validation Result, like sending notifications based on the validation's outcomes.
 
-<InProgress/>
+[Batch Definition](/core/connect_to_data/connect_to_data.md): A configuration for how a Data Asset should be divided into individual batches for testing.
+
+[Checkpoint](/core/trigger_actions_based_on_results/create_a_checkpoint_with_actions.md): An object that groups Validation Definitions and runs them with shared parameters and automated Actions.  Checkpoints are the primary means for validating data in a production deployment of Great Expectations.
+
+[Data Asset](/core/connect_to_data/connect_to_data.md): A collection of records within a Data Source which are usually grouped based on the underlying data system and can be further sliced into Batches according to a desired specification.
+
+[Data Context](/core/set_up_a_gx_environment/create_a_data_context.md): A Data Context stores all of the configuration for a project in Great Expectations, and provides an API to access and update it.
+
+[Data Docs](/core/configure_project_settings/configure_data_docs/configure_data_docs.md): Human readable documentation generated from Great Expectations metadata detailing Expectations, Validation Results, etc.
+
+[Data Source](/core/connect_to_data/connect_to_data.md): An object that tells GX how to access a specific source of external data and provides a unified API within GX for organizing that data into Data Assets which can be retrieved. 
```

Comment:
Consider s/access a specific source of external data/connect to a specific source of external data/.